classes/class-wc-connect-nux.php

```diff
@@ -93,6 +93,14 @@ public function show_pointers( $hook ) {
 			wp_enqueue_script( 'wc_services_admin_pointers' );
 		}
 
+		public function get_dismissed_pointers() {
+			$data = get_user_meta( get_current_user_id(), 'dismissed_wp_pointers', true );
+			if ( is_string( $data ) && 0 < strlen( $data ) ) {
```

Comment:
We can return this here:

```
explode( ',' get_user_meta( get_current_user_id(), 'dismissed_wp_pointers', true ) );
```

In that case ^, I'm not sure we need a separate function.

Background: the two checks aren't required because`$data` is always a string and `explode($data)` will return an array with a single empty string if `$data` is an empty string. `implode` will return an empty string for both an empty array and an array with a item that is an empty string:

```
implode(array()); // ''
implode(array('')); // ''
```

And `in_array` will work as usual.